lib/grpc_reflection/service/builder/util.ex

```diff
@@ -7,6 +7,22 @@ defmodule GrpcReflection.Service.Builder.Util do
 
   @type_message Map.fetch!(Google.Protobuf.FieldDescriptorProto.Type.mapping(), :TYPE_MESSAGE)
 
+  def get_nested_types(symbol, descriptor), do: get_nested_types(symbol, descriptor, [])
+
+  def get_nested_types(_symbol, %Google.Protobuf.DescriptorProto{nested_type: []}, acc) do
+    acc
+  end
+
+  def get_nested_types(symbol, %Google.Protobuf.DescriptorProto{nested_type: nested_types}, acc) do
+    Enum.reduce(nested_types, acc, fn nested_type, acc ->
```

Comment:
nit: you are recursing _and_ reducing here?  It seems to work, but usually you only see one at a time as they are different approaches to the same problem